src/Glpi/Api/HL/Controller/CoreController.php

```diff
@@ -462,6 +462,7 @@ public function token(Request $request): Response
         } catch (OAuthServerException $exception) {
             return $exception->generateHttpResponse(new JSONResponse());
         } catch (\Throwable $exception) {
+            trigger_error($exception->getMessage(), E_USER_WARNING);
```

Comment:
This would permit to have the stack trace in the logs.
```suggestion
            \Glpi\Application\ErrorHandler::getInstance()->handleException($exception, true);
```